main.js

```diff
@@ -129,3 +132,38 @@ autoUpdater.on('update-downloaded', (ev, info) => {
 app.on('ready', function()  {
   autoUpdater.checkForUpdates();
 });
+
+function fixMeasureUnit(value, type, precision) {
```

Comment:
Sure, I've been there ;)

Like this? https://github.com/iffy/electron-updater-example/pull/17/commits/aef3887b508bbd1fe2987d9f23c1d53f1e2c35dc